src/CovidSim.cpp

```diff
@@ -59,6 +59,8 @@ int GetInputParameter(FILE*, FILE*, const char*, const char*, void*, int, int, i
 int GetInputParameter2(FILE*, FILE*, const char*, const char*, void*, int, int, int);
 int GetInputParameter3(FILE*, const char*, const char*, void*, int, int, int);
 
+void setICDF(double* icdf, int startValue);
```

Comment:
Function name should be SetICDF to conform with the other PascalCase global function names.

Reply:
Agreed - change committed.